packages/core/next-env.d.ts

```diff
@@ -1,5 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
+/// <reference types="next/navigation-types/compat/navigation" />
```

Comment:
question: what does it mean? Was it generated automatically?

Reply:
I think that's something regarding the compatibility with the new next/navigation (previously was router). This changes was generated when I run `yarn dev`.